frontend/src/components/Calendar/ClickableDateContainer.tsx

```diff
@@ -33,21 +32,21 @@ const EventIndicator = styled.div<{ selected?: boolean; eventPresent?: boolean }
   background-color: ${(props) => (props.selected ? 'white' : '#468751')};
 `
 
-export const ClickableDateContainer = (props: { date: Date; eventPresent?: boolean; facilityId: number }) => {
+export const ClickableDateContainer = (props: {
+  date: Date
+  eventPresent?: boolean
```

Comment:
would hasEvent be a better name? and do consider making a type for this instead! ie 
`type Pros = {...}`